src/main/java/org/eclipse/cargotracker/interfaces/booking/facade/BookingServiceFacade.java

```diff
@@ -28,4 +30,8 @@
 	List<Location> listShippingLocations();
 
 	List<CargoRoute> listAllCargos();
+
+	List<TrackingId> listAllTrackingIds();
+
+	CargoTracking loadCargoForTracking(String trackingId);
```

Comment:
Probably want to call this something like CargoStatus. CargoTracking is too generic.